services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/validation/HankeValidator.kt

```diff
@@ -0,0 +1,45 @@
+package fi.hel.haitaton.hanke.validation
+
+import fi.hel.haitaton.hanke.Hanke
+import fi.hel.haitaton.hanke.HankeError
+import org.springframework.validation.Errors
+import org.springframework.validation.Validator
+import javax.validation.ConstraintValidator
+import javax.validation.ConstraintValidatorContext
+
+class HankeValidator : ConstraintValidator<ValidHanke, Hanke> {
+
+    private fun checkInputString(input: String?): Boolean {
+        return input == null || input.trim { it <= ' ' }.length == 0
```

Comment:
Sopisiko tähän kotlinin oma String?.isNullOrBlank-funktio? https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/is-null-or-blank.html
Muutenkin voisi olla luettavampaa tehdä extension-funktio, jos tarvitsee tällaisia omia funktioita tehdä.